Backend/Tumultu/Infrastructure/Infrastructure.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
How about .net 8? xd 👉👈

Reply:
When I was starting project it was still unavailable for me, will change soon.